nyc_taxis/TEST_PROCEDURES.md

```diff
@@ -0,0 +1,54 @@
+# Test Procedures
+
+## searchable_snapshot 
```

Comment:
Could you add documentation about the supported OpenSearch versions for this feature?

Reply:
Thank you so much for your review! @engechas 👍👍 I will address the comments one by one.

Reply:
Added the supported version in line 11 (https://github.com/opensearch-project/opensearch-benchmark-workloads/pull/58/files#diff-a3c7b0ad3c2082729bd2ee692decaa774510b3ffa044732d4e37d8445d1f2cc1R11). It's a good idea to explicitly mention the version since because the feature is not supported from the beginning.